client.go

```diff
@@ -78,6 +78,16 @@ type (
 		//	- InternalServiceError
 		GetWorkflowHistory(workflowID string, runID string) (*s.History, error)
 
+		// GetWorkflowThreadDump gets thread dump of a particular workflow.
+		// The errors it can return:
+		//	- EntityNotExistsError
+		//	- BadRequestError
+		//	- InternalServiceError
+		GetWorkflowThreadDump(workflowID string, runID string) (string, error)
```

Comment:
nit: How about renaming this to GetWorkflowStackTrace() ?  ThreadDump can be misleading in Golang.